clients/shared/append.go

```diff
@@ -21,10 +21,12 @@ func Append(dwh destination.DataWarehouse, tableData *optimization.TableData, op
 		return fmt.Errorf("failed to get table config: %w", err)
 	}
 
+	cols, _ := columns.RemoveDeleteColumnMarker(tableConfig.Columns().GetColumns())
+
 	// We don't care about srcKeysMissing because we don't drop columns when we append.
 	_, targetKeysMissing := columns.Diff(
-		tableData.ReadOnlyInMemoryCols(),
-		tableConfig.Columns(),
+		tableData.ReadOnlyInMemoryCols().GetColumns(),
```

Comment:
Actually, shouldn't we exclude it from the source and not target?

Isn't this going to add a column `__artie_deleted` in the destination?

Reply:
Ooh yeah good catch!